pkg/linter/linterExecute.go

```diff
@@ -65,17 +65,27 @@ func computeConnectivity(configs map[string]*vpcmodel.VPCConfig) (map[string]*vp
 	return nodesConn, nil
 }
 
-// //////////////////////////////////////////////////////////////////////////////////////////////
-// LinterExecute executes linters one by one
+// LinterExecute performs the lint analysis and then prints the string result; should be redundant once lint is
+// integrated in the general flow
 func LinterExecute(configs map[string]*vpcmodel.VPCConfig, printAllFindings bool,
 	enableList, disableList []string) (issueFound bool, resString string, err error) {
-	nodesConn, err := computeConnectivity(configs)
+	linters, err := linterAnalysis(configs, enableList, disableList)
 	if err != nil {
 		return false, "", err
 	}
+	resString = linters.String(printAllFindings)
+	fmt.Println(resString)
+	return issueFound, resString, nil
```

Comment:
Where do you set `issuesFound`?

Reply:
its not, and not really need. removed (the function was supposed to be deleted before PR I did not bother in the first place)